src/token/mod.rs

```diff
@@ -13,583 +13,440 @@
 //! [signing](sign_access_token) and [verifying](verify_access_token) access tokens.
 //!
 //! **NOTE: The APIs in this module are experimental and likely to change in the future!**
-//! This is because we plan to move much of the code here to the [coset](https://docs.rs/coset/)
+//! This is because we plan to move much of the code here to the [`coset`]
 //! library, since much of this just builds on COSE functionality and isn't ACE-OAuth specific.
 //!
-//! In order to use any of these methods, you will need to provide a cipher which handles
-//! the cryptographic operations by implementing either [`CoseEncryptCipher`],
-//! [`CoseMacCipher`] or [`CoseSignCipher`], depending on the intended operation.
+//! In order to use any of these methods, you will need to provide a cryptographic backend which
+//! handles the cryptographic operations by implementing either [`EncryptCryptoBackend`],
+//! [`MacCryptoBackend`](cose::MacCryptoBackend) or [`SignCryptoBackend`], depending on the intended
```

Comment:
Because the access token module currently does not provide methods for creating MAC-ed access tokens, and `MacCryptoBackend` is therefore not already imported into this module.